otel-agent/ecs-ec2/config.yaml

```diff
@@ -63,20 +68,28 @@ exporters:
     application_name: "$APP_NAME"
     subsystem_name: "$SUB_SYS"
     application_name_attributes:
+    - "aws.ecs.cluster"
+    - "aws.ecs.cluster.name"
     - "aws.ecs.task.definition.family"
     subsystem_name_attributes:
+    - "aws.ecs.container.name"
+    - "aws.ecs.docker.name"
```

Comment:
Not sure if valid. Seems not documented at https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor#amazon-ecs
Maybe just use "aws.ecs.task.family"?